src/extension.ts

```diff
@@ -270,7 +275,7 @@ function addSelectedLinesStatusBarItem(context: vscode.ExtensionContext) {
     context.subscriptions.push(selectedLinesStatusBarItem);
 
     // register some listener that make sure the status bar 
-    // item and the currently opened file always up-to-date
+    // item and the sidebar always up-to-date
```

Comment:
@jurekseverin-isg  
Here the entry in the changelog is missing.

Reply:
Added this at [here](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/72)